src/components/date-range-picker/date-range-picker.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { Component } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { FormsModule } from '@angular/forms';
+import { PopoverModule } from '../../components/popover';
+import { IconModule } from '../icon';
+import { DateRangePickerModule } from './date-range-picker.module';
+
+
+@Component({
+    selector: 'ux-components-date-range-picker',
+    template: `
+        <ng-template #popoverTemplate>
```

Comment:
You will need to remove <code><ng-template></code> from this. Regarding the karma tests should maybe add a few more tests to test the inputs eg <code>start</code>, <code>end</code> and the outputs eg <code>startChange</code>, <code>endChange</code>

Reply:
Agreed, the e2e tests are good for testing based on input to the text area as this is not part of the component, however it would be good to add karma tests to actually test the component inputs and outputs are functioning as expected.

Reply:
updated tests and added tests for start, end and onRangeChange